src/openms/source/ANALYSIS/DECHARGING/FeatureDeconvolution.cpp

```diff
@@ -1039,8 +1039,7 @@ namespace OpenMS
     FeatureMapType fm_missing;
     cl.suggestMissingFeatures(fm_out, cons_map, fm_missing);
 
-    FeatureXMLFile fmf;
-    fmf.store("fm_missing.featureXML", fm_missing);
+    FileHandler.storeFeatures("fm_missing.featureXML", fm_missing);
```

Comment:
maybe worth to dicuss: is it a better API for a user to have this moved to a FileHandler class? Wouldn't it be more natural to have such a method in FeatureMap? Separation is still be possible if I am not mistaken.

Reply:
or e.g., keeping the old interface with FeatureXMLFile etc. that uses some internal FeatureXMLFileHandler

Reply:
> e.g., keeping the old interface with FeatureXMLFile, so code doesn't break and call some internal FeatureXMLFileHandler

this is what we do for `MzMLFile` and `MzMLFileHandler` and what I implemented here in this PR as well, separating the interface `FeatureXMLFile` and `FeatureXMLFileHandler`. But the question that makes sense is whether we need 2 interfaces, `FileHandler` and `MzMLFile`

Reply:
Ok I see. I agree - having a FeatureXMLFile interface looks a bit cleaner designwise compared to having only a "FileHandler" class. But no strong opinion here.

Reply:
But let's keep the two interfaces for now